tests/tasks/setup_candlepin.yml

```diff
@@ -3,6 +3,17 @@
 - name: Setup the test data
   import_tasks: setup_test_data.yml
 
+- name: Install needed packages
+  package:
+    name:
+      - podman
+    state: present
+    use: "{{ __rhc_is_ostree |
+              ternary('ansible.posix.rhel_rpm_ostree', omit) }}"
+
+- name: Remove Candlepin container
+  include_tasks: teardown_candlepin.yml
+
```

Comment:
Hm there is no need to install podman when running the tests using an external registration server, especially that it takes many seconds.

Apologies for my previous suggestion, let's move this back in the block.